### PR TITLE
#6/overflow in home and profile screen fixed

### PR DESCRIPTION
**Pull request by YuvrajSingh3110**
Issue #6 fixed with fixing the overflow in home screen and UI issue in profile screen

### CODE REVIEW

File: lib/view/nav_bar/home_screen.dart [hunk 1]
```diff
@@ -87,105 +84,101 @@ class _HomeState extends State<HomeScreen> {
...
+                                  return Text('Error: ${snapshot.error}');
+                                }
 
-                                  if (!snapshot.hasData) {
-                                    return Container(
-                                        color: Colors.white,
-                                        child: const Center(
-                                            child: SpinKitFadingCube(
-                                                color: primary, size: 100.0)));
-                                  }
```
Comment (Photon3009) [inline lib/view/nav_bar/home_screen.dart]: Avoid making changes for unnecessary indentation

**Review by Photon3009 — CHANGES_REQUESTED** [review]
I have tested your changes, and the homescreen is still getting overflowed by 13 pixels. This means that it is still device-specific. As for the profile screen, I recommend making adjustments so that we don't have to make the content scrollable. This can be achieved by adjusting the font size and padding using MediaQuery.
Try to update code as per the suggestions and I will convert this PR from easy to medium level. So, you still have 2 more days.

**Comment by YuvrajSingh3110** [comment]
please review the changes if the issue still persists or not

**Review by Photon3009 — APPROVED** [review]

**Comment by Photon3009** [comment]
@YuvrajSingh3110  Good work! I am merging this branch, but the profile screen is still device-specific. I recommend creating a new issue for that.

**Pull request merged**